### PR TITLE
Document ssl_verify option for AutoYaST

### PR DESCRIPTION
**Pull request by cjschroder**
re jira#SLE-21833

### PR creator: Description

How to disable SSL verification for HTTPS installation paths
Reviewers, see attached PDF

[invoking-autoinst_color_en.pdf](https://github.com/SUSE/doc-sle/files/7514678/invoking-autoinst_color_en.pdf)

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 SP4/openSUSE Leap 15.4 *(current `main`, no backport necessary)*
  - [x] SLE 15 SP3/openSUSE Leap 15.3
  - [x] SLE 15 SP2/openSUSE Leap 15.2
  - [x] SLE 15 SP1
  - [x] SLE 15 GA
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4
  - [ ] SLE 12 SP3

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Review by imobachgs — APPROVED** [review]
It looks good to me. Thanks!

File: xml/ay_auto_installation.xml [hunk 1]
```diff
@@ -280,6 +280,19 @@ xml:id="Invoking">
        Location of the installation directory, for example
        <literal>install=nfs://192.168.2.1/CDs/</literal>.
       </para>
+      <note>
+       <title>Disabling SSL checks</title>
+       <para>
+         When you are using HTTPS, SSL checking is enabled by default. If 
+         necessary, you can disable SSL checking by appending <literal>?ssl_verify=no</literal>
```
Comment (lslezak) [inline xml/ay_auto_installation.xml:287]: Um, actually appending the `?ssl_verify=no` text is not correct in all cases, if there already is an option in the URL you should append the `&ssl_verify=no` text. The `?` character separates the URL query from path, the `&` character separates multiple query options. That's the usual [URL format](https://en.wikipedia.org/wiki/URL).

E.g. `https://example.com?foo=bar` => `https://example.com?foo=bar&ssl_verify=no`

So I'd rephrase it as

```
... by appending the <literal>ssl_verify=no</literal> query option ...
```

Comment (cjschroder) [reply xml/ay_auto_installation.xml]: Good catch, thanks!

**Comment by cjschroder** [comment]
SLE15SP3 b5ef7e781, 7a2888c87

**Comment by cjschroder** [comment]
SLE15SP2 86c20879e 52608b83a
SLE15SP1 03fb5d45b
SLE15SP0 586986336
